javaparser-core/src/main/java/com/github/javaparser/ast/body/AnnotationMemberDeclaration.java

```diff
@@ -137,25 +137,29 @@ public SimpleName getName() {
      * @return this, the AnnotationMemberDeclaration
      */
     public AnnotationMemberDeclaration setDefaultValue(Expression defaultValue) {
+        notifyPropertyChange("defaultValue", this.defaultValue, defaultValue);
```

Comment:
I really think we should start generating parts of the AST. Will look into that after generating visitors.